modules/mobile/spec/services/sync_update_service_spec.rb

```diff
@@ -1,26 +1,45 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
+require_relative '../support/helpers/rails_helper'
 
 describe Mobile::V0::Profile::SyncUpdateService do
-  let(:user) { create(:user, :api_auth) }
   let(:service) { Mobile::V0::Profile::SyncUpdateService.new(user) }
 
   before do
-    allow(Flipper).to receive(:enabled?).with(:mobile_v2_contact_info, instance_of(User)).and_return(false)
+    allow(Flipper).to receive(:enabled?).with(:mobile_v2_contact_info, instance_of(User)).and_return(true)
     allow(Flipper).to receive(:enabled?).with(:va_v3_contact_information_service, instance_of(User)).and_return(false)
     allow(Flipper).to receive(:enabled?).with(:remove_pciu, instance_of(User)).and_return(false)
   end
 
-  # DO THIS
-  describe '#save_and_await_response', :skip_va_profile_user do
-    let(:params) { build(:va_profile_address, vet360_id: user.vet360_id, validation_key: nil) }
+  describe '#v2_save_and_await_response' do
+    before do
+      Flipper.enable(:va_v3_contact_information_service)
+    end
+
+    after do
+      Flipper.disable(:va_v3_contact_information_service)
+    end
+
+    let(:user) { create(:user, :api_auth_v2) }
+
+    let(:params) { build(:va_profile_v3_address, :v2_override, validation_key: nil) }
```

Comment:
The response address does not match the va_profile_v3_address, :v2_override address. I would assume this could be the case for many of these tests. The test should be failing.